src/controllers/loginController.ts

```diff
@@ -57,6 +58,29 @@ const validatingUser = async (req: Request, res: Response) => {
 
       res.cookie('token', token);
       res.header('Authorization', `Bearer ${token}`);
+
+      // Send verification email if the User is not verified yet
+
+      const verificationToken = jwt.sign({ email: user.email }, secret, {
+        expiresIn: '15m',
+      });
+
+      user.verificationToken = verificationToken;
+      await user.save();
+      const { email } = user;
+      const { name } = user;
+
+      const mailOptions = {
+        to: email,
+        subject: 'Your Email verification',
+        template: 'email',
+        context: {
+          name,
+          verificationLink: `http://localhost:3000/api/users/verify-email/${verificationToken}`,
+        },
```

Comment:
this can happened when user didn't verfiy his/her info and no loggin should happened